stock_picking_account_date/models/stock_move.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021, Jarsa Sistemas, S.A. de C.V.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lpgl.html).
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _prepare_account_move_vals(
+        self,
+        credit_account_id,
+        debit_account_id,
+        journal_id,
+        qty,
+        description,
+        svl_id,
+        cost,
+    ):
+        if self.picking_id.accounting_date:
+            context = dict(self.env.context)
+            context["force_period_date"] = self.picking_id.accounting_date
+            self = self.with_context(**context)
+        return super()._prepare_account_move_vals(
+            credit_account_id,
+            debit_account_id,
+            journal_id,
+            qty,
+            description,
+            svl_id,
+            cost,
+        )
+
+    def _get_price_unit(self):
+        if self.picking_id.accounting_date and self.purchase_line_id:
+            po_line = self.purchase_line_id
+            order = po_line.order_id
+            price_unit = po_line.price_unit
+            if po_line.taxes_id:
+                price_unit = po_line.taxes_id.with_context(round=False).compute_all(
```

Comment:
Mmmh, I'm not sure recomputing price from forced accounting date is the good solution.

However, this should be reflected either in README, either in field's help and explain why this is done.